README.md

```diff
@@ -58,72 +57,107 @@ Input data can be found on the ESIP S3 Bucket (see "Accessing Data through ESIP
-- `params_template.env`
+
+There are two ways, which can be used together, to configure the system and/or data processing. Some configuration is based on input arguments when running `fim_pipeline.sh` described below in the "Produce HAND Hydrofabric" section. Another configuration option is based on using a file named `params_template.env`, found in the `config` directory. To use this latter technique, copy the `params_template.env` file before editing and remove the word "template" from the filename. The `params.env` file includes, among other options, a calibrated parameter set of Manning’s n values. The new `params.env` becomes one of the arguments submitted when running `fim_pipeline.sh`.
+
+Make sure to set the config folder group to 'fim' recursively using the chown command.
 
 This system has an optional tool called the `calibration database tool`. In order to use this system, you have three options:  
-1.  Install the calibration database service.
+1.  Install the calibration database service, found in the [calibration tools README.md](https://github.com/NOAA-OWP/inundation-mapping/tree/dev/tools/calibration-db).
 2.  Disable it by providing the `-skipcal` command line option to `fim_pipeline.sh` or `fim_preprocessing.sh`.
-3.  Disable it in the `params_template.env` file. See [calibration tool README](https://github.com/NOAA-OWP/inundation-mapping/blob/dev/tools/calibration-db/README.md) for more details.
+3.  Disable it in the `params.env` file. See [calibration tool README](https://github.com/NOAA-OWP/inundation-mapping/blob/dev/tools/calibration-db/README.md) for more details.
 
 ### Start/run the Docker Container
 
-Since all of the dependencies are managed in utilizing a Docker container, we must issue the [`docker run`](https://docs.docker.com/engine/reference/run/#clean-up---rm) command to start a container as the run-time environment. The container is launched from a Docker Image which was built in [Installation](#installation) step 2. The correct input file pathing is necessary for the `/data` volume mount (`-v`) for the `<input_path>`. The `<input_path>` should contain a subdirectory named `/inputs` (similar to `s3://noaa-nws-owp-fim/hand_fim`). If the pathing is set correctly, we do not need to adjust the `params_template.env` file, and can use the default file paths provided.
+Since all of the dependencies are managed by utilizing a Docker container, we must issue the [`docker run`](https://docs.docker.com/engine/reference/commandline/run/) command to start a container as the run-time environment. The container is launched from a Docker image which was built in [Installation](#installation). The `-v <input_path>:/data` and must contain a subdirectory named `inputs` (similar to `s3://noaa-nws-owp-fim/hand_fim`). If the pathing is set correctly, we do not need to adjust the `params_template.env` file, and can use the default file paths provided.
```

Comment:
"The `-v <input_path>:/data`  must contain" reads more smoothly than  

"The `-v <input_path>:/data` and must contain"